packages/imitation_game/README.md

```diff
@@ -0,0 +1,89 @@
+# Imitation Game
+
+## Description
+
+`imitation_game` is a platform for performing automated tests to compare Flutter
+to alternative platforms.  For example, how much memory does the same app
+written in Flutter and UIKit take?
+
+## Running all the tests
+
+You need a mobile device plugged into your computer and setup for development.
+The mobile device and the computer need to be on the same network, one that
+allows communication between computers since that's how the mobile phone will
+report its results to the computer.
+
+```sh
+dart imitation_game.dart
+```
+
+## Dependencies
+
+In order to run the tests you will need the union of all the platforms being
+tested.  As new tests are added please add to this list:
+
+### iOS
+
+- Flutter
+- Xcode
+- [ios_deploy](https://github.com/ios-control/ios-deploy) - used to launch apps
+  on the attached iOS device.
+
+## File Layout
+
+```
+./
+├─ imitation_game.dart
+└─ tests/
+   ├─ smiley/
+   │  ├─ README.md
+   │  ├─ flutter/
+   │  │  └─ run_ios.sh
```

Comment:
I don't know how well that is going to scale.  The tests take a long time to execute since compiling and launching on a device takes so long.  Seems like we should have the best implementation per platform.  If there were 2 different approaches it might be better to name the implementation accordingly (ex `flutter_with_fast_library`).

Reply:
Yes, we should only maintain the best implementation and the imitation goal implementation in the long run, and only use this extra level for a few new implementations to be uploaded and tested temporarily. They'll either be eliminated or replace the old best implementation. I also agree that the name should be more informative. Sorry for my bad naming examples 😄